scenarios/bin/onos-cmd

```diff
@@ -2,12 +2,7 @@
 # SPDX-FileCopyrightText: 2020-present Open Networking Foundation <info@opennetworking.org>
 # SPDX-License-Identifier: Apache-2.0
 
-if [ -z $1 ]; then
-  echo "usage: $0 cmd [args...]"
-  exit 1
-fi
-
-# Use sshpass to skip the password prompt
-docker run -it --rm --network host ictu/sshpass \
-  -procks ssh -o "UserKnownHostsFile=/dev/null" \
-  -o "StrictHostKeyChecking=no" -o LogLevel=ERROR -p 8101 onos@localhost "$@"
+node=$1
+cliCmd=$2
+output=$3
+onos "$node" "$cliCmd" >> "$output"
```

Comment:
any reason to change this? This would affect the `set-app-info` and `set-app-debug` make target in the `scenarios/Makefile`

Reply:
Nope. Just for ease of development. Reverted to the original script